common/services/prismic/opening-times.ts

```diff
@@ -239,12 +248,12 @@ export function getVenueById(venues: Venue[], id: string): Venue | undefined {
 export function getTodaysVenueHours(
   venue: Venue
 ): ExceptionalOpeningHoursDay | OpeningHoursDay | undefined {
-  const todaysDate = london();
-  const todayString = formatDay(todaysDate.toDate());
+  const todaysDate = today();
```

Comment:
Today() is declared as
```
// This is just to allow us to mock values in tests
export function today(): Date {
  return new Date();
}
```
but this isn't a test, right? I see it gets "timezoned" through `todayString`, but maybe we could rename that to indicate where it happens?

Reply:
That function being used outside of mock values for tests is what confused me about the comment, when I thought it was _only_ used for tests.